modules/core/src/test/scala/GraphQLResponseTests.scala

```diff
@@ -9,16 +9,16 @@ import munit.Assertions._
 
 object GraphQLResponseTests {
   def assertWeaklyEqual(x: Json, y: Json, strictPaths: List[List[String]] = Nil): Unit =
-    assert(weaklyEqual(x, y, strictPaths))
+    assert(weaklyEqual(clue(x), clue(y), strictPaths))
```

Comment:
Anything wrapped in `clue` will be printed when the assertion fails, useful for debugging.